crates/re_viewer/src/ui/selection_panel.rs

```diff
@@ -309,13 +321,13 @@ fn blueprint_ui(
                         // TODO(emilk): show the values of this specific instance (e.g. point in the point cloud)!
                     } else {
                         // splat - the whole entity
-                        let space_view_class_name = *space_view.class_name();
+                        let space_view_class = *space_view.class_name();
```

Comment:
Why rename here? The previous one was better imo.

Reply:
Yeah I went back and forth with these and this one clear got forgotten.